python/src/trezorlib/cli/benfen.py

```
@@ -0,0 +1,90 @@
+
+    resp = benfen.sign_tx(
+        client, 
+        address_n, 
+        decoded_message,
+        decoded_coin_type
+    )
+    result = {
+        "public_key": f"0x{resp.public_key.hex()}",
+        "signature": f"0x{resp.signature.hex()}",
+    }
+    return result
```

Ruff (0.8.2)

Unnecessary assignment to `result` before `return` statement

Remove unnecessary assignment

(RET504)